BLT/BLT/UserProfileViewController.swift

```diff
@@ -56,30 +66,54 @@ class UserProfileViewController: UIViewController
         userImage.clipsToBounds = true
         userImage.layer.cornerRadius = 50
         
-        userDataChart.clipsToBounds = true
-        userDataChart.layer.cornerRadius = 25
-        userDataChart.backgroundColor = .gray
+//        userDataChart.clipsToBounds = true
```

Comment:
Remove comments for code that has been replaced

Reply:
Fixed in commit 3155623.